src/ui/component/filePicker/filePicker.tsx

```diff
@@ -42,14 +37,11 @@ export const FilePicker: FC<FilePickerProps> = ({
     state === 'dragging-over' ? setIsDraggingOver(true) : setIsDraggingOver(false)
   }, [])
 
-  const toFileDTO = useCallback(
-    (file: globalThis.File) => ({
-      name: file.name,
-      fullPath: file.webkitRelativePath,
-      size: file.size,
-      mediaType: file.type,
-      stream: file.stream()
-    }),
+  const toFileDto = useCallback(
+    (file: File): FileDTO =>
+      Object.assign(file, {
+        path: file.webkitRelativePath ? file.webkitRelativePath : `/${file.name}`
+      }),
```

Comment:
If you don't mind, I prefer to not build the `path` here and let the consumer handle it from the returned `File` object; this ton respect the contract between the generic component and its caller.

Reply:
Sure I'll make the necessary changes :) It makes more sense anyway